types/yaireo__tagify/index.d.ts

```diff
@@ -1602,6 +1608,31 @@ declare class Tagify<T extends Tagify.BaseTagData = Tagify.TagData> {
      */
     setDisabled(disabled: boolean): void;
 
+    /**
+     * Get data for the specific instance by `key` parameter from `localStorage`.
+     * @param key `localStorage` key (under the tagify namespace).
+     * @returns Data stored under `key` in `localStorage`.
+     * Returns `undefined` if {@link TagifyConstructorSettings.id} has not been set or no entry exists for `key` in `localStorage`.
+     */
+    getPersistedData(key: string): any;
+
+    /**
+     * Set data for the specific instance.
+     * Must supply a second parameter which will be the key to save the data in the `localStorage`
+     * (under the tagify namespace).
+     * In order to use this method, {@link TagifyConstructorSettings.id} must be set.
+     * @param data Data to store in `localStorage`.
+     * @param key `localStorage` key (under the tagify namespace).
+     */
+    setPersistedData(data: any, key: string): void;
```

Comment:
Same as above, for consistency `data` should be of type `unknown` (although it doesn't matter much for parameters)